aspnetcore/includes/net-core-prereqs-vsc-2.2.md

```diff
@@ -1,3 +1,3 @@
 * [Visual Studio Code](https://code.visualstudio.com/download)
 * [.NET Core SDK 2.2 or later](https://www.microsoft.com/net/download/all)
-* [C# for Visual Studio Code version 1.17.1 or later](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
\ No newline at end of file
+* [C# for Visual Studio Code version 1.19.1 or later](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
```

Comment:
Guaranteed to be out of date 99% of the time. How about

[C# for Visual Studio Code current version](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)